doc/source/api/index.rst

```diff
@@ -3,7 +3,41 @@
 =============
 API Reference
 =============
-Here we should discuss the different API modules at a high level.
+
+This section describes the core pythonic interfaces for Fluent. 
+Here, you can find the application programming interfaces for 
+such things as launching Fluent, assorted utilities, as well as 
+the interfaces for the meshing and solver components of Fluent. 
```

Comment:
This seems to duplicate [the wording above](https://github.com/pyansys/pyfluent/pull/593/files#diff-91eae658e0456633fa9d05405629a93ec359ecfc472898a356b87c94b733bf35)

Reply:
Yes, because the contents of the API Reference page and the Core page refer to the same material. It used to have more but now it seems redundant to have this extra level. We can address this organizational issue at a later stage, right?